src/Interpreters/ClusterProxy/executeQuery.cpp

```diff
@@ -517,14 +517,11 @@ void executeQueryWithParallelReplicas(
                 "`cluster_for_parallel_replicas` setting refers to cluster with several shards. Expected a cluster with one shard");
     }
 
-    auto coordinator = std::make_shared<ParallelReplicasReadingCoordinator>(
```

Comment:
Let remove the include for coordinator from this file

Reply:
Ok,thanks